client/src/components/DataDisplay/TheTotalTraffic.vue

```diff
@@ -0,0 +1,17 @@
+<template>
+    <base-traffic-card title="Total Traffic" :traffic="totalTraffic" />
```

Comment:
you can just do `mapGetter` in this component

Reply:
@daniel-pupczyk-rmtcfm-com I don't know if it's good solution. This component only displays data passed by props and I'm using it in 3 different files in many places. So using getters in this component isn't good solution, because one time I want to pass totalTraffic (one getter), second time I want to pass traffic only for homepage (second getter), etc.